jekyll/_cci2/sample-config.md

```diff
@@ -776,6 +776,7 @@ jobs:
           name: Build iperf3
           working_directory: iperf
           command: |
+            CIRCLE_WORKING_DIRECTORY=$(eval "echo $CIRCLE_WORKING_DIRECTORY")
             IPERF3_MAKE_PREFIX=$CIRCLE_WORKING_DIRECTORY/<< parameters.label >>
```

Comment:
Would `../<< parameters.label >>` work instead?

Reply:
ya, but I think this here to demonstrate the need to use "eval" to expand the working directory into a usable path.

Reply:
The only other reference is here in discuss: https://discuss.circleci.com/t/circle-working-directory-doesnt-expand/17007

Reply:
oh hah! i only quickly looked at this and thought it was part of the build config for the docs themselves 🤦